integration-libs/opf/base/core/services/opf-endpoints.service.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import {
+  BaseSiteService,
+  DynamicAttributes,
+  StringTemplate,
+} from '@spartacus/core';
+
+import { OpfApiConfig, OpfConfig } from '@spartacus/opf/base/root';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OpfEndpointsService {
+  private _activeBaseSite: string;
+
+  constructor(
+    protected opfConfig: OpfConfig,
+    protected opfApiConfig: OpfApiConfig,
+    protected baseSiteService: BaseSiteService
```

Comment:
[MEDIUM]
Do we really need the config injected 2 times? Isn't it the same object in the end of the day?